### PR TITLE
 rke2: cilium: Update the chart to 1.10.402

### PR DESCRIPTION
**Pull request by vadorovsky**
https://github.com/cilium/cilium/releases/tag/v1.10.4

Ref: rancher/rke2#1615

Signed-off-by: Michal Rostecki <mrostecki@opensuse.org>

### CODE REVIEW

**Review by briandowns — DISMISSED** [review]

**Review by dereknola — DISMISSED** [review]

**Review by brandond — CHANGES_REQUESTED** [review]
This needs to go in a new version section - see 
https://github.com/rancher/kontainer-driver-metadata/pull/652#issuecomment-915457725

You may want to just wait and rebase this once that PR is merged.

**Comment by vadorovsky** [comment]
@brandond Ok. Since the new version is added in the PR you linked, I will just put this one on hold.

**Comment by vadorovsky** [comment]
@brandond should be ready now

**Review by brandond — APPROVED** [review]

**Comment by deniseschannon** [comment]
Based on this PR, I think this could be closed? https://github.com/rancher/kontainer-driver-metadata/pull/693

**Comment by dereknola** [comment]
Yes this PR has been superseded by https://github.com/rancher/kontainer-driver-metadata/pull/693

**Pull request closed without merging**
